analysis.qmd

```diff
@@ -6,48 +6,62 @@ The draft currently has no official status. It is a work in progress and is subj
+
+The Commissioner’s role is to review any changes in design or methodology that the Analyst brings to their attention. The Commissioner should be available to provide input and clarifications to the Analyst. The aim is for the Commissioner to be aware of and accept changes to the design or methodology providing they meet their requirements. 
+
+### The Approver's responsibilities during the analysis phase
+
+The Approver should be aware of the progress of the analysis and ensure that they are available for approving the work at the Delivery phase.
+
+## Assurance during the analysis phase
+
+### Verification and validation
 
-The analytical assurer shall review the assurance completed by the analyst, carry out any further validation and verification they may see as appropriate, and report errors and areas for improvement to the analyst. The assurer may then need to re-review the assurance completed, as required. 
+TBC
 
-The assurer shall also confirm the work has been appropriately scoped, executed, validated, and verified and documented to the approver. 
+### Handling Uncertainty during the analysis phase
 
-## The Commissioner's responsibilities during the analysis phase
+While the Scoping and Design phases identified and described risks and uncertainties, the Analysis phase aims to assess and quantify the impact of uncertainty on the analytical outcome and their contribution to the range and likelihoods of possible outcomes. Guidance on how to identify, mitigate, measure and communicate uncertainty can be found in the [Uncertainty Toolkit for Analysts](https://analystsuncertaintytoolkit.github.io/UncertaintyWeb/index.html). The verification and validation by the analyst and assurer should assure the appropriate treatment of uncertainty.
 
-The commissioner’s role is to review the analysis and assurance as fit for the purposes of the commission.
+## Multi-use models and the analysis phase
 
-## The Approver's responsibilities during the analysis phase
+In multi-use models, analysis and edits may be carried out on individual elements of the model at differing times. This calls for mechanisms for assuring that the changes integrate into the larger model as expected, for example, through the use of test-suites.
 
-The Approver scrutinises the work of the analyst and assurer and shall be confident that the analysis:
+## Artificial Intellingence and Machine Learning (Black Box) models and the analysis phase
 
-* Meets its design requirements;
-* Is of sufficient quality and
-* Is adequately documented.
+AI and ML models are not as transparent as traditionally coded models. This adds challenge to the assurance of these models as compared to other forms of analysis. Assurance activities during this phase include performance testing and formal verification. See the [Introduction to AI Assurance](https://www.gov.uk/government/publications/introduction-to-ai-assurance) for further details.
 
-## Uncertainty
+## Working with third parties
```

Comment:
On working with third parties - academics, contractors etc. Think we need to define what we mean by a third party in the definitions sector. The bit we haven't covered is the act of commissioning the work. We have a Commissioner who is basically the customer and then what normally happens is an analysis speicfies the work for a thrid party. Thus there is an intermeidary. 
Seems to me the important pointhere is that the specification is clear about who performs each role i.e. is the third party acting as the Analyst or the Analyst and Assurer or...
Do we need a  name for the person specifying the work  for the third party?